sources/europe/lv/lv_orthophoto_cycle1_1994-1999.geojson

```diff
@@ -2,7 +2,7 @@
     "type": "Feature",
     "properties": {
         "id": "ORTOFOTO_V1",
-        "name": "Orthophoto 1st cycle (1994-1999) - Latvia",
+        "name": "Orthophoto (1994–1999), B&W, 1:30000, Latvia",
         "type": "wms",
         "url": "https://services.lgia.gov.lv/arcfree/services/Ortofoto_v1/MapServer/WmsServer?LAYERS=0&STYLES=&FORMAT=image/jpeg&CRS={proj}&WIDTH={width}&HEIGHT={height}&BBOX={bbox}&VERSION=1.3.0&SERVICE=WMS&REQUEST=GetMap",
```

Comment:
Would it possible to chagne the url to "https://services.lgia.gov.lv/arcfree/services/WMS_Ortofoto_v1/MapServer/WmsServer?..."

Reply:
As long as you're sure that it works. I have no idea who to check that WMS service works. Have you checked the 6th cycle orthophoto?

Reply:
In most cases the checks are enough, but in this case there seems to be an issue with the WMS server. The server does not follow the WMS 1.3.0 specification and does not invert the x,y coordinates for the bbox parameter for the EPSG:4326 coordinate system.
 
The following parameters work:

```
 "url": "https://lvmgeoserver.lvm.lv/geoserver/ows?width={width}&height={height}&bbox={bbox}&format=image/vnd.jpeg-png8&request=GetMap&service=WMS&styles=&transparent=true&version=1.1.1&srs={proj}&layers=public:Orto_LKS",
```
  
```     
"available_projections": [
            "CRS:84",
            "EPSG:3059",
            "EPSG:3857",
            "EPSG:4326"
        ],
```

Reply:
I will change at once to the values you wrote, correct?

Reply:
Done.